### ISSUE TITLE
[EN][Quest][Uldum] Tipping the Balance

### ISSUE THREAD

**Issue by MnmZilla** [Atlantiss/BugTracker#8095]
I never get into the cinematic in this quest, and then if I try talking to the Harrison Jones NPC he just spawns another copy of himself which jumps on top of the statue, which then later disappears. If I try to talk to him again he just repeats the same thing.

http://cata.openwow.com/quest=27431

**Comment by MnmZilla** [comment]
Could you please explain what I did wrong the last time I opened this issue? I am afraid that, to my knowledge, I followed the instructions in the 'readme' correctly.

**Comment by Aival94** [comment]
you speak to wrong npc. The one you speak is the one who is supposed to spawn when the event end. There is one NPC on the ground(not on this cave) to start the event.

**Comment by MnmZilla** [comment]
Waited for 2-3minutes, do I need to wait more?

**Comment by Silverdevilboy** [comment]
Same issue here.

Activating the quest spawns a Harrison model which runs over to the colossus, jumps up twice and appears to get stuck under the skirt-thing the colossus wears. Harrison either stays fixed there falling, or stops and stands on the back of its calf before despawning in a couple seconds, and then the colossus despawns a couple minutes after that.

He only says the one line before disappearing, just as in the screenshot above.

**Comment by Pappoli** [comment]
I don't know if this matters but, as an alliance player, I have the same problem

**Comment by cosminush** [comment]
there is a problem with npc from image. Something "kill" the cinematic.
image: http://imgur.com/a/cK5UK

**Comment by maslak** [comment]
bump.

**Comment by Danbar1976** [comment]
still not working

**Comment by worloq** [comment]
pls fix it, it still doesn't work

**Comment by Silverdevilboy** [comment]
Its kinda sad, between this quest and the prisoner quest, half of uldum is unplayable right now.

**Comment by Eltreakol** [comment]
Unless the quests after this do not work, this is a quest that opens up most of the quests in Uldum, including those that need item drops (Scnottz Landing). Maybe make [Efficient Excavations] (Item that drops from Officers in the Obelisk of the Sun) to start the Scnottz Landing quest line drop regardless of if you have done this quest?

**Comment by Sayris9876** [comment]
Trying this for 20 min already clicking on Harrison Jones like crazy :D Nothing works

**Comment by karhima** [comment]
Doesn't work, been there with 2 other players trying to get it to work. This blocks half of Uldum as already has been said. There's not alternative if you like doing quests. Please prioritise.

**Comment by Silverdevilboy** [comment]
It works. It's rare. Have done it since the last update. Had to kill it 20 times, but it drops.

On 1 Oct 2016 2:27 p.m., karhima <notifications@github.com> wrote:

Doesn't work, been there with 2 other players trying to get it to work. This blocks half of Uldum as already has been said. There's not alternative if you like doing quests. Please prioritise.

**Comment by karhima** [comment]
There's nothing to kill, no idea what you're talking about.

**Comment by Silverdevilboy** [comment]
Ignore comments before, the email came in a thread about a different quest for reasons unknown.

**Comment by Silverdevilboy** [comment]
A dragon spawns and flies down when you use the flag. Takes about 30s. Have some patience.

On 1 Oct 2016 2:49 p.m., karhima <notifications@github.com> wrote:

There's nothing to kill, no idea what you're talking about.

**Comment by Anhaeyn** [comment]
Still not working.

**Comment by artamedan** [comment]
I don't know what happened here but this quest and ALL the Harrison Jones quest-line worked in March 2016. I remember that you had to wait a while for this quest to be completed, not mounted and the scene-cut worked fine.
I have all 105 Uldum quests done.

**Comment by Sayris9876** [comment]
Yes dude it worked fine i did uldum in all of my previous characters but latest one this quest was bugged and apparently its still is.

**Comment by DataMemoryStick** [comment]
Same here , it's **bugged** can't complete it. Used to work a while back.

**Comment by Digglewiggle** [comment]
still bugged

**Comment by Fuzzypunk** [comment]
still bugged

**Comment by maslak** [comment]
bump!!!

**Comment by Thorghar** [comment]
Still BUGGED...

**Comment by Silverdevilboy** [comment]
Worked for me on an orc warrior. Still doesn't work on an elf paladin. Might be like other quests that only work once upon server restart.

**Comment by Aival94** [comment]
Not sure but i did the quest just fine. I unmount,took quest,speak with npc and chooce the option and then follow the event npc(the quest npc does not disappear) and completed the quest just fine.

**Comment by Sayris9876** [comment]
Maybe is one of those that can be done once per server restart? Since for some people is bugged for others working. I dont know...

**Comment by Anhaeyn** [comment]
Today when I was leveling alt, did that quest with no problems, but there was server restart like 1 hour earlier, soo maybe that's why it worked this time.
